### PR TITLE
Add Og::getRole helper method

### PR DESCRIPTION
**Pull request by amitaibu**
Split out of #268

### CODE REVIEW

**Comment by amitaibu** [comment]
PR also adds the missing `OgRoleInterface::getId()`

**Comment by amitaibu** [comment]
Ready for review

**Comment by jhedstrom** [comment]
This makes sense to me, does it make sense to test in one of the existing tests?

File: src/Entity/OgRole.php [hunk 1]
```diff
@@ -37,6 +37,16 @@
 class OgRole extends Role implements OgRoleInterface {
 
   /**
+   * Gets the ID of the role.
+   *
+   * @return string
+   *   The unique ID for the role.
+   */
+  public function getId() {
```
Comment (amitaibu) [inline src/Entity/OgRole.php]: actually it's not needed, as we have $role->id(). I'll remove.

**Comment by amitaibu** [comment]
> does it make sense to test in one of the existing tests?

It does. I'll add one.

**Comment by amitaibu** [comment]
Obviously adding the one liner test found a bug in  my code :) And then I realized there's a simpler way to write the method - it's just a wrapper over `OgRole::load()`.

**Comment by jhedstrom** [comment]
👍

**Pull request merged**